### PR TITLE
Added TemperatureIR, Tilt, DistanceUS, Distance IR, Voltage/Current, Sound Intensity, Linear Poti, Rotary Poti, Barometer, Hall Effect and Piezo Speaker

### PR DESCRIPTION
**Pull request by lucas-grab**
I added Nodes for the TemperatureIR, Tilt and DistanceUS Bricklets

### CODE REVIEW

File: bricklets/DistanceUS.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
+<!--
+  Copyright 2016 IBM Corp.
```
Comment (hardillb) [inline bricklets/DistanceUS.html]: You should probably update this to be 2017

File: bricklets/Tilt.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+        node.ipcon.on(Tinkerforge.IPConnection.CALLBACK_CONNECTED,
+        function(connectReason) {
+            node.t = new Tinkerforge.BrickletTilt(node.sensor, node.ipcon);
+
+            node.interval = setInterval(function(){
+                if (node.t) {
+                    node.t.getTiltState(function(tiltState) {
+
+					//	if(tiltState === 0) {
```
Comment (hardillb) [inline bricklets/Tilt.js]: Should this be commented out?

**Review by hardillb — CHANGES_REQUESTED** [review]
Mostly looks ok, needs the Copywrite dates updating in all files and look at the tilt sensor output

File: bricklets/DistanceUS.html (hunk 1 above)
Comment (lucas-grab) [reply bricklets/DistanceUS.html]: done

File: bricklets/Tilt.js (hunk 2 above)
Comment (lucas-grab) [reply bricklets/Tilt.js]: Wasn't sure if the output should be just the numbers or also an explanation.
Changed it to just the numbers, the explanation is in the info section of the node.

File: bricklets/Tilt.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        function(connectReason) {
+            node.t = new Tinkerforge.BrickletTilt(node.sensor, node.ipcon);
+
+            node.interval = setInterval(function(){
+                if (node.t) {
+                    node.t.getTiltState(function(tiltState) {
+
+                        node.send({
+                            topic: node.topic || 'tilt',
+                            payload: "tiltState"
```
Comment (hardillb) [inline bricklets/Tilt.js]: You are still just sending the string "tiltState" here not the value

Comment (lucas-grab) [reply bricklets/Tilt.js]: You're right. Fixed it.

**Comment by lucas-grab** [comment]
@hardillb I also added a Node for the DistanceIR Node.

**Comment by lucas-grab** [comment]
@hardillb I also added Voltage/Current, Sound Intensity, Linear Poti, Rotary Poti, Barometer, Hall Effect and Piezo Speaker.  Please take a look :)

**Comment by hardillb** [comment]
Just to be clear, you do have examples of all these sensors you've added to test them with?

**Review by hardillb — APPROVED** [review]

**Comment by lucas-grab** [comment]
@hardillb Yes, I did test them with the sensors.

**Comment by lucas-grab** [comment]
@hardillb Added the Line bricklet. Did you have a chance to look at the code?

**Pull request merged**